src/server_manager/web_app/app.ts

```diff
@@ -609,21 +608,34 @@ export class App {
 
   private showTransferStats(runningServer: server.Server, serverView: Polymer) {
     const refreshTransferStats = () => {
-      runningServer.getDataUsage().then((stats) => {
-        // Calculate total bytes transferred.
-        let totalBytes = 0;
-        // tslint:disable-next-line:forin
-        for (const accessKeyId in stats.bytesTransferredByUserId) {
-          totalBytes += stats.bytesTransferredByUserId[accessKeyId];
-        }
-        serverView.setServerTransferredData(totalBytes);
-        // tslint:disable-next-line:forin
-        for (const accessKeyId in stats.bytesTransferredByUserId) {
-          const transferredBytes = stats.bytesTransferredByUserId[accessKeyId];
-          const relativeTraffic = totalBytes ? 100 * transferredBytes / totalBytes : 0;
-          serverView.updateAccessKeyRow(accessKeyId, {transferredBytes, relativeTraffic});
-        }
-      });
+      runningServer.getDataUsage().then(
+          (stats) => {
+            // Calculate total bytes transferred.
+            let totalBytes = 0;
+            // tslint:disable-next-line:forin
+            for (const accessKeyId in stats.bytesTransferredByUserId) {
+              totalBytes += stats.bytesTransferredByUserId[accessKeyId];
+            }
+            serverView.setServerTransferredData(totalBytes);
+            // tslint:disable-next-line:forin
+            for (const accessKeyId in stats.bytesTransferredByUserId) {
+              const transferredBytes = stats.bytesTransferredByUserId[accessKeyId];
+              const relativeTraffic = totalBytes ? 100 * transferredBytes / totalBytes : 0;
+              serverView.updateAccessKeyRow(accessKeyId, {transferredBytes, relativeTraffic});
+            }
+          },
+          (e) => {
+            // Since failures are invisible to users we generally want exceptions here to bubble
+            // up and trigger a Sentry report. The exception is network errors, about which we can't
+            // do much (note: ShadowboxServer generates a breadcrumb for failures regardless, for
+            // anyone who explicitly submits feedback).
+            if (e instanceof errors.ServerApiError) {
+              if (!e.response) {
```

Comment:
Let's add a `IsNetworkError()` to make it more explicit. `e.response` is not meaningful.

Reply:
This helps make things a little more readable here. Done.